examples/default/src/screens/LegacyModeScreen.tsx

```diff
@@ -0,0 +1,117 @@
+import React, { useState } from 'react';
+import { ActivityIndicator } from 'react-native';
+import Instabug from 'instabug-reactnative';
+import { ListTile } from '../components/ListTile';
+import { Screen } from '../components/Screen';
+import { showNotification } from '../utils/showNotification';
+
+export const LegacyModeScreen: React.FC = () => {
+  const [loading, setLoading] = useState(false);
+
+  const addInstabugLog = (log: string): Promise<void> => {
+    return new Promise<void>((resolve) => {
+      Instabug.logDebug(log);
+      resolve();
+    });
+  };
```

Comment:
There's no need for the `new Promise(...)` in these functions I think, we can just simplify them to something like this:
```suggestion
  const addInstabugLog = (log: string) => {    
      Instabug.logDebug(log);
  };
```